OpenAL32/alSource.cpp

```diff
@@ -54,6 +54,9 @@ using namespace std::placeholders;
 
 inline ALvoice *GetSourceVoice(ALsource *source, ALCcontext *context)
 {
+    if(!source || !context)
```

Comment:
These can't be null, since they should already have been checked (if not used) before the call. If they manage to be null is some situation, it needs to be detected earlier so the appropriate error can be set before getting to this point.